Version_testNet

```diff
@@ -0,0 +1,3 @@
+DOCKER_IMAGE_FUSE_APP_VERSION="1.0.0"
```

Comment:
Can you add an empty line for all the files where it's missing? @Andrew-Pohl

Reply:
Not sure I get you here like a change in EOL characters (should be unix LF), or do you mean a trailing new line at EOF?